[Bug]: telepathy-glib build failed

### Problem description

/home/builder/.termux-build/_cache/android-r25b-api-24-v0/bin/../sysroot/usr/include/stdio.h:137:6: note: 'rewind' declared here
void rewind(FILE* __fp);
     ^
test-tpl-log-walker.c:382:20: error: too many arguments to function call, expected single argument '__fp', have 3 arguments
  rewind (fixture, walker, 3);
  ~~~~~~           ^~~~~~~~~
/home/builder/.termux-build/_cache/android-r25b-api-24-v0/bin/../sysroot/usr/include/stdio.h:137:6: note: 'rewind' declared here
void rewind(FILE* __fp);
     ^
test-tpl-log-walker.c:384:20: error: too many arguments to function call, expected single argument '__fp', have 3 arguments
  rewind (fixture, walker, 1);
  ~~~~~~           ^~~~~~~~~
/home/builder/.termux-build/_cache/android-r25b-api-24-v0/bin/../sysroot/usr/include/stdio.h:137:6: note: 'rewind' declared here
void rewind(FILE* __fp);
     ^
test-tpl-log-walker.c:401:20: error: too many arguments to function call, expected single argument '__fp', have 3 arguments
  rewind (fixture, walker, 8);
  ~~~~~~           ^~~~~~~~~
/home/builder/.termux-build/_cache/android-r25b-api-24-v0/bin/../sysroot/usr/include/stdio.h:137:6: note: 'rewind' declared here
void rewind(FILE* __fp);
     ^
test-tpl-log-walker.c:403:20: error: too many arguments to function call, expected single argument '__fp', have 3 arguments
  rewind (fixture, walker, 8);
  ~~~~~~           ^~~~~~~~~
/home/builder/.termux-build/_cache/android-r25b-api-24-v0/bin/../sysroot/usr/include/stdio.h:137:6: note: 'rewind' declared here
void rewind(FILE* __fp);
     ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make[4]: *** [Makefile:826: test-tpl-log-walker.o] Error 1
make[4]: Leaving directory '/home/builder/.termux-build/telepathy-glib/src/tests/logger/dbus'
make[3]: *** [Makefile:762: all-recursive] Error 1
make[3]: Leaving directory '/home/builder/.termux-build/telepathy-glib/src/tests/logger'
make[2]: *** [Makefile:1246: all-recursive] Error 1
make[2]: Leaving directory '/home/builder/.termux-build/telepathy-glib/src/tests'
make[1]: *** [Makefile:531: all-recursive] Error 1
make[1]: Leaving directory '/home/builder/.termux-build/telepathy-glib/src'
make: *** [Makefile:461: all] Error 2
builder@5b3c90e51e03:~/termux-packages$
[telepathy-glib.txt](https://github.com/termux/termux-packages/files/9685322/telepathy-glib.txt)


### What steps will reproduce the bug?

build

### What is the expected behavior?

_No response_

### System information

termux-info:

```

```

--- a/x11-packages/telepathy-glib/build.sh
+++ b/x11-packages/telepathy-glib/build.sh
@@ -2,11 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://telepathy.freedesktop.org/
 TERMUX_PKG_DESCRIPTION="GLib bindings for the Telepathy D-Bus protocol"
 TERMUX_PKG_LICENSE="LGPL-2.1"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.99.11
-TERMUX_PKG_REVISION=7
-TERMUX_PKG_SRCURL=https://telepathy.freedesktop.org/releases/telepathy-glib/telepathy-glib-$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=268cbf0199804ecb6001c7c11e5596bc3cea40a600091c144d14f837ac90cd97
-TERMUX_PKG_DEPENDS="dbus-glib"
+# Do not bump to 0.99.x.
+TERMUX_PKG_VERSION=1:0.24.2
+TERMUX_PKG_SRCURL=https://telepathy.freedesktop.org/releases/telepathy-glib/telepathy-glib-${TERMUX_PKG_VERSION:2}.tar.gz
+TERMUX_PKG_SHA256=b0a374d771cdd081125f38c3abd079657642301c71a543d555e2bf21919273f0
+TERMUX_PKG_DEPENDS="dbus-glib, glib"
 TERMUX_PKG_BREAKS="telepathy-glib-dev"
 TERMUX_PKG_REPLACES="telepathy-glib-dev"
 TERMUX_PKG_BUILD_IN_SRC=true

